tests/plans/test_export.py

```diff
@@ -156,3 +157,8 @@ def test_reply_to_mixin(plan_factory, project_factory, administrative_district):
 
     assert plan.created.astimezone().isoformat() == export.get_created_data(plan)
     assert plan.modified.astimezone().isoformat() == export.get_modified_data(plan)
+    if plan.is_draft:
```

Comment:
As we create the plan above, we also know if it is draft or not. The usual factory one isn't. https://github.com/liqd/a4-meinberlin/blob/58aa85d2e5fea07841b63ab6cbb11f98e6c384b9/meinberlin/test/factories/plans.py#L29

You could test another one that is and have both cases tested. But let's ask @CarolingerSeilchenspringer first how important the yes/no is, when we do use the False/True for other bools.

Reply:
Well, @Rineee also looked at it and is right: we do already add a lot of complicated export stuff here, so one more and exactly as the story says is also fine. I will quickly look at the test, so that both cases are tested.